mkdocs.yml

```diff
@@ -237,6 +237,7 @@ nav:
       - installation.md
       - releases.md
       - upgrading.md
+      - features-3.6.md
```

Comment:
I don't think this should be a separate page per version as this will just keep increasing and make it hard to see other docs in this section. It's also inconsistent with the [upgrading guide](https://argo-workflows.readthedocs.io/en/release-3.5/upgrading/) and CHANGELOG, both of which are single pages (with CHANGELOG being eventually split into two per https://github.com/argoproj/argo-workflows/pull/13352). Deprecations from #13735 is also a single page.

We previously discussed this together in the Oct 1st Contributor Meeting or earlier if I recall correctly, both before and after I met with CNCF Tech Writers

Reply:
We can drop new `features-3.6.md` from 3.7 because they'll be in 3.6's documentation. This is partially to enable automation of features-3.7.md too.

I can't find other projects that have a mega page full of new features for all prior versions, it seems normal to have one page per feature release.

Reply:
Renamed to `new-features.md`